integrations/plugin-debugger/src/JovoDebugger.ts

```diff
@@ -464,15 +464,6 @@ export class JovoDebugger extends Plugin<JovoDebuggerConfig> {
     await this.emitDebuggerConfig();
     await this.emitLanguageModelIfEnabled();
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    function propagateStreamAsLog(stream: Writable, socket: typeof Socket) {
```

Comment:
Please add this function again. It is not used now but might be used in the future.

Reply:
Added it in `utilities.ts` :+1:

Reply:
Can you remove the `@ts-ignore` please? Casting would be a much cleaner solution if necessary.